scripts/installer/install.py

```diff
@@ -269,8 +269,8 @@
    # 1d. check java version
    java_version = subprocess.check_output(["java", "-version"], stderr=subprocess.STDOUT).decode()
    print("Found java version "+java_version)
-   if not re.search("1.8", java_version):
-      sys.exit("Dataverse requires Java 1.8. Please install it, or make sure it's in your PATH, and try again")
+   if not re.search('(1.8|11|14)', java_version):
+      sys.exit("Dataverse requires Java 1.8, 11 or 14. Please make sure it's in your PATH, and try again.")
```

Comment:
Supporting Java 14 feels a little arbitrary. It's not an LTS release. It just happens to be the current release. My concern is that if we really want to support the latest release of Java, we're going to have to edit this line every six months. I'd be happier with just putting the next LTS release in there, Java 17, which is due out September 2021. Or we could remove Java 14.

Here's a table from Wikipedia that shows all the recent Java releases:

<img width="253" alt="Screen Shot 2020-06-15 at 2 10 42 PM" src="https://user-images.githubusercontent.com/21006/84691526-4deef700-af12-11ea-83ec-818687f21880.png">

Reply:
@donsizemore we talked about this a bit in Slack. Can you please remove Java 14 and only support 8 and 11? Thanks!